precompiles/erc20/integration_test.go

```diff
@@ -393,6 +396,7 @@ var _ = Describe("ERC20 Extension -", func() {
 				// NOTE: we are not passing the contract call here because this test is for direct calls only
 
 				// FIXME: This error message currently is different from the EVM extension message
+				// -- says "ERC20: transfer amount exceeds balance" instead of "spendable balance ... is smaller than ..."
 				Entry(" - through erc20 contract", erc20Call),
```

Comment:
The standard errors given by the ERC20 implementation are very generic IMO, rather than just mirror them directly, can't we wrap them with more info?

Reply:
These errors are improved by the errors in ERC-6093, which contain more information when parsed. Current task is to parse them correctly.

Reply:
ERC-6093 will be handled in a follow-up PR 👍